master.tf

```diff
@@ -23,29 +23,37 @@ resource "linode_instance" "k8s_master" {
     kernel = "linode/direct-disk"
 
     devices {
-      sda = {
+      sda {
         disk_label = "boot"
       }
     }
   }
 
   provisioner "file" {
-    source      = "${path.module}/scripts/"
+    source      = "${path.cwd}/${path.module}/scripts/"
     destination = "/tmp"
 
     connection {
-      user    = "core"
-      timeout = "300s"
+      host        = self.ip_address
+      agent       = "false"
```

Comment:
In the CLI, we go to great lengths to ensure that the user is properly using an ssh agent. Can you switch this back to true and test it out?

Reply:
Sure thing, testing this now.

Reply:
This works with agent = "true" and I have updated all instances of this.